go/vt/vtgate/planbuilder/testdata/unsupported_cases.json

```diff
@@ -392,11 +392,6 @@
     "query": "select distinct a, b as a from user",
     "plan": "generating order by clause: ambiguous symbol reference: a"
   },
-  {
```

Comment:
this query didn't make much sense to me, so I removed it. The problem is with `in (select *` - `IN` works against a single column. I think we acceptable coverage of star expansion on system tables even without this query